### ISSUE TITLE
Installation on Ubuntu 15.10 fails

### ISSUE THREAD

**Issue by renozao** [prey/prey-node-client#197]
I believe it is the same error as #173

Installation fails and ends with the following:
```
Running post_activate hooks...
Voila! All good.
--
Installing init scripts.
Sweet! Please run `prey config gui` to finish installation.
Voila! All good.
No protocol specified

(gksu:2660): Gtk-WARNING **: cannot open display: :0.0
dpkg: error processing package prey (--install):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 prey
```

I can solve this by doing `xhost +local:root`.

But the GUI does not start although it says:

```
Firing up config GUI!
All done!
```

Then I can only run the configuration using `python2` but not as sudo, which prevents from accessing the configuration files. 
So overall quite not working as intended.

Is this a known issue?
Thanks

**Comment by deepdivingspearo** [comment]
I can confirm this behaviour on Ubuntu Gnome 15.10.

**Comment by raliste** [comment]
Thank you for reporting. Already working on a fix.

Reported by #129, #198, #197, #194

**Comment by javo** [comment]
Fixed on 1.6.1 Prey version.
